src/xdocs/config_javadoc.xml

```diff
@@ -2518,27 +2516,27 @@ public class Test extends Exception {
         </p>
         <source>
 &lt;module name=&quot;SummaryJavadocCheck&quot;&gt;
-  &lt;property name=&quot;period&quot; value="。"/&gt;
```

Comment:
The full stop symbol replaced with its `&#` value to keep the Java file pure ASCII.